tests/test_pitch_txt_parser.py

```diff
@@ -0,0 +1,114 @@
+from unittest import mock
+
+import song2txt.uspitch.uspitch.txt_parser as parser
+
+import pytest
+import numpy as np
+
+
+@pytest.fixture
+def text():
+    text = '#TITLE:title\n' \
+           '#MP3:filename.mp3\n' \
+           '#BPM:300\n' \
+           '#GAP:10,5\n' \
+           '* 0 4 58 ña\n' \
+           '- 8 10\n' \
+           ': 12 5 58 ña\n' \
+           ': 17 2 55 ~\n' \
+           '- 20\n' \
+           'E\n'
+    return text
+
+
+@pytest.fixture
+def metadata():
+    metadata = {'TITLE': 'title',
```

Comment:
This seems extracted from the previous one, you should probably do it automatically following some automatic procedure or you'll have to change both manually. Generally, there should be a single source of truth for test data.